xds/src/main/java/io/grpc/xds/GrpcXdsTransportFactory.java

```diff
@@ -35,34 +36,45 @@
 final class GrpcXdsTransportFactory implements XdsTransportFactory {
 
   static final GrpcXdsTransportFactory DEFAULT_XDS_TRANSPORT_FACTORY =
```

Comment:
Move this into the tests that use it? It isn't used any more in production. (I probably wouldn't make it a CONSTANT like it is here, at it is no longer the DEFAULT and such. I'd probably just create a new instance when needed.)